spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/JdbcAggregateJobQueryDao.java

```diff
@@ -665,6 +681,21 @@ private List<TaskJobExecution> getTaskJobExecutionsByDate(Date startDate, Date e
 				endDate
 		);
 	}
+	private class JobInstanceExtractor implements ResultSetExtractor<List<JobInstance>> {
+
+		@Override
```

Comment:
I think this can be simplified a bit, something like...
```java
		@Override
		public List<JobInstance> extractData(ResultSet rs) throws SQLException, DataAccessException {
			List<JobInstance> jobInstances = new ArrayList();
			while (rs.next()) {
				jobInstances.add(new JobInstance(rs.getLong("JOB_INSTANCE_ID"), rs.getString("JOB_NAME")));
			}
			return jobInstances;
		}
```
Wdyt?